index.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+const args = process.argv.slice(2);
+const {Plop, run} = require('plop');
+const argv = require('minimist')(args);
```

Comment:
minimist package is not available in dependencies.

Reply:
It is covered in this PR - https://github.com/pagesource/component-generator/pull/3/files